P/ParseGen/build_tarballs.jl

```diff
@@ -9,19 +9,19 @@ version = v"2.0.0"
 sources = [
     ArchiveSource("https://github.com/sandialabs/parsegen-cpp/archive/refs/tags/v$(version).tar.gz", "c6c7c4958d1c6ab77bf0970b5aacae4b63603f702666492638ee8c0bdf3125c8"),
     ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.15.sdk.tar.xz",
-    "2408d07df7f324d3beea818585a6d990ba99587c218a3969f924dfcc4de93b62")
+    "2408d07df7f324d3beea818585a6d990ba99587c218a3969f924dfcc4de93b62"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/parsegen-cpp*/
 
 install_license LICENSE
-
 mkdir build && cd build
 
 if [[ "${target}" == x86_64-apple-darwin* ]]; then
     export CXXFLAGS="-mmacosx-version-min=10.15"
```

Comment:
You likely don't need this line. And the entire aarch64-apple-darwin branch is now useless

Reply:
This isn't resolved. I'm still convinced this line isn't needed.

Reply:
Thanks, didn't see the line you were referring to the first time.